### ISSUE TITLE
Regional usage vs. support data discrepancy

### ISSUE THREAD

**Issue by NOtherDev** [Fyrd/caniuse#1939]
I've noticed that the JSON data available in this repository are not consistent in regards of browsers versions, when there is one only. 

For example, let's look at the first feature file: https://github.com/Fyrd/caniuse/blob/master/features-json/aac.json - it has a single `9.9` version key for `and_uc` browser. Meanwhile, the regional usage data, i.e. here: https://github.com/Fyrd/caniuse/blob/master/region-usage-json/AD.json names the single key for `and_uc` just `0`. The pattern is consistent through all the regions and all the single-version browsers available.

**Comment by Fyrd** [comment]
Yeah...this is because the usage data is generally only updated once a month, and for these "0" values all usage of the browser is given to that one version (due to lacking a version breakdown of usage information for these browsers).

I suppose the solution would be to update the regional data when any such version changes. Will consider that.
